### ISSUE TITLE
symfony composer conflict

### ISSUE THREAD

**Issue by corneliusio** [solspace/craft-calendar#169]
**Describe the bug or issue you're experiencing**
We're currently trying to get our clients upgraded to Craft 4 and have run into an issue on one of our sites where the craft-calendar plugin requirements are causing conflicts with a number of other plugins. Specifially with the following:
* `hybridinteractive/craft-contact-form-extensions` > `albertcht/invisible-recaptcha` > `illuminate/view` > `illuminate/filesystem` > `symfony/finder` 
* `hybridinteractive/craft-contact-form-extensions` > `albertcht/invisible-recaptcha` > `illuminate/view` > `illuminate/collections`. 

I think that specifically, this is a result of the version requirements for the `symfony` packages. I was able to get things working by adding `^6.0` to those restrictions: https://github.com/solspace/craft-calendar/commit/8ed21d3e422400a86c72fce96c510308ad75b14b

**Steps to reproduce**

`composer.json`
```json
{
    "require": {
        "hybridinteractive/craft-contact-form-extensions": "4.2.0",
        "solspace/craft-calendar": "4.0.0-beta.8"
    },
    "config": {
        "allow-plugins": {
            "craftcms/plugin-installer": true,
            "yiisoft/yii2-composer": true
        }
    }
}
```

**Expected behavior**
Both plugins should be able to be installed.

**Craft & Plugin Info (please complete the following information):**
Above dependencies.

- Craft Version: Multiple (But specific site we're working on is 4.2.1.1)
- Calendar Version: 4.0.0-beta.8
- Calendar Edition: Pro
- Fresh Install or Upgrade: Both

**Comment by kjmartens** [comment]
Sorry about this @corneliusio. We'll get this fixed shortly. 🙂

**Comment by kjmartens** [comment]
This should now be resolved in v4.0.0. 🙂
